UnityProject/Assets/Scripts/ChemistryComponents/ReagentContainer.cs

```diff
@@ -204,7 +211,7 @@ public void ReagentsChanged(bool applyChange = true)
 			if (ReactionSet != null)
 			{
 				possibleReactions.Clear();
-				foreach (var Reagents in currentReagentMix.reagents.m_dict)
+				foreach (var Reagents in CurrentReagentMix.reagents.m_dict)
```

Comment:
```suggestion
				foreach (var reagent in CurrentReagentMix.reagents.m_dict)
```
isn't this a single reagent?